dependencies.props

```diff
@@ -38,11 +35,6 @@
       <RawVersionsBaseUrl>https://raw.githubusercontent.com/dotnet/versions</RawVersionsBaseUrl>
     </DependencyBuildInfo>    
 
-    <XmlUpdateStep Include="BuildTools">
```

Comment:
What's the set of things you're still using from Buildtools in this repo?

Reply:
I'm not sure, but I know that it's small. Most likely very few tasks, we probably depend on some properties set by Build.Common.props/targets, etc. My next work item is to try to cut Standard off from Buildtools entirely.